### PR TITLE
ripmime: add page

### PR DESCRIPTION
**Pull request by pandyah5**
- [x] The page(s) are in the correct platform directories: `linux`
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): v1.4.0.9**

### CODE REVIEW

File: pages/linux/ripmime.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- Extract file contents in a specific directory:
+
+`ripmime -i {{path/to/file}} -d {{path/to/dir}}`
+
+- Extract file contents and print verbose output:
+
+`ripmime -i {{path/to/file}} -v`
+
+- To get detailed information about the whole decoding process:
```
[the remark this answers is not in the document]
Comment (pandyah5) [reply pages/linux/ripmime.md]: Sure! Resolved in [4c621b0](https://github.com/tldr-pages/tldr/pull/11654/commits/4c621b09003e401fe1d086bd5fd47a51a7371a8b)

File: pages/linux/ripmime.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# ripmime
+
+> Extract attachments out of a MIME encoded email packages.
```
[the remark this answers is not in the document]
Comment (pandyah5) [reply pages/linux/ripmime.md]: Sure! Resolved in [4c621b0](https://github.com/tldr-pages/tldr/pull/11654/commits/4c621b09003e401fe1d086bd5fd47a51a7371a8b)

**Review by kbdharun — APPROVED** [review]
LGTM, Welcome to tldr and thanks for your contribution. I have a minor suggestion, then this PR is GTG.

For the path placeholder, I would suggest keeping it verbose unless the command hits the 250 character limit.

File: pages/linux/ripmime.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+> Extract attachments out of a MIME encoded email package.
+> More information: <https://pldaniels.com/ripmime>.
+
+- Extract file contents in the current directory:
+
+`ripmime -i {{path/to/file}}`
+
+- Extract file contents in a specific directory:
+
+`ripmime -i {{path/to/file}} -d {{path/to/dir}}`
```
[the remark this answers is not in the document]
Comment (pandyah5) [reply pages/linux/ripmime.md]: Sure @kbdharun, I have made the changes!

**Pull request merged**